src/containers/AccountConnection.jsx

```diff
@@ -258,8 +265,11 @@ class AccountConnection extends Component {
               messages={[t('account.message.error.global.description', {name: connector.name})]}
             /> }
 
-            { !success && <KonnectorSync
+            { (connector.accounts.length > 0 && !success && !error) && <KonnectorSync
```

Comment:
If you rebase on master you will replace `connector.accounts.length > 0` by `editing`, and remove occurences l.270 and in `KonnectorSync` l.9